plugins/sonarqube/src/components/index.ts

```diff
@@ -15,3 +15,4 @@
  */
 
 export * from './SonarQubeCard';
+export { isSonarQubeAvailable as isPluginApplicableToEntity } from './useProjectKey';
```

Comment:
Actually, we are migrating away from the `isPluginApplicableToEntity` pattern to `isSonarQubeAvailable`. So you can just export it without an alias.

See https://backstage.io/docs/plugins/composability#naming-patterns

Reply:
Got it, I've just pushed a commit to drop the alias. Also, thanks for including the reference, that's useful.